lib/quby/questionnaires/entities/questionnaire.rb

```diff
@@ -183,9 +181,20 @@ def to_codebook(options = {})
             output << ""
           end
 
+          flags.each do |_flag_key, flag|
```

Comment:
Idd lelijk, maar het is nu (helaas) de conventie. Lijkt me fijner om het in een aparte refactor-PR te doen zodat we één standaard houden